nengo/neurons.py

```diff
@@ -231,11 +231,13 @@ class LIFRate(NeuronType):
 
     tau_rc = NumberParam('tau_rc', low=0, low_open=True)
     tau_ref = NumberParam('tau_ref', low=0)
+    amplitude = NumberParam('amplitude', low=0)
```

Comment:
I wonder if this should be `low_open=True`? I suppose one could set the amplitude to 0 without crashing Nengo, but it seems like it wouldn't make much sense.

Also, this parameter needs to be added to the docstring.